qiskit/transpiler/passes/scheduling/instruction_alignment.py

```diff
@@ -10,24 +10,23 @@
 # copyright notice, and modified files need to carry a notice indicating
 # that they have been altered from the originals.
 
-"""Align measurement instructions."""
-import itertools
+"""Analysis passes for hardware alignment constraints."""
```

Comment:
could you also put the 3 passes in this file into separate files? I'm fine if you want to put them in a subdirectory of `passes/scheduling/alignment`. (and maybe all the padding ones in the other PR under `passes/scheduling/padding`. (Or keep them all flat under `passes/scheduling`, up to you)

Reply:
Done in https://github.com/Qiskit/qiskit-terra/pull/7762/commits/2e542aeffe76abdc6ff5714c9a4539d5e8b27d9b